apps/3000-home/pages/[...slug].js

```diff
@@ -0,0 +1,9 @@
+import { injectScript } from '@module-federation/utilities'
+import { createDynamicFederatedPage } from '@module-federation/nextjs-mf/utils'
+
+const { Page, getServerSideProps } = createDynamicFederatedPage({
+  injectScriptReplacement: injectScript
```

Comment:
Wasn't able to find how to fix import issue for `injectScript`, so temporarily I inject it from the outside